repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -175,8 +195,14 @@ def version2_check(info):
     for instance in info.instances:
         if _manifest_get(instance.manifest, 'release', None) == '1.00':
             continue
-        if not _fullfills_hana_min_version(instance):
-            _add_hana_details(found, instance)
+        if version.matches_target_version('> 8.6', '> 9.0'):
```

Comment:
```suggestion
        if version.matches_target_version('> 8.6', '< 9.0') or version.matches_target_version('> 9.0', '< 10.0'):
```

Reply:
any target release  >9.0 should return warning in the report, so it's safe to skip '<10.0', right?

Reply:
yes, it shouldyes, it should.